premailer/premailer.py

```diff
@@ -114,7 +114,8 @@ def __init__(self, html, base_url=None,
                  cache_css_parsing=True,
                  cssutils_logging_handler=None,
                  cssutils_logging_level=None,
-                 disable_leftover_css=False):
+                 disable_leftover_css=False,
+                 image_align=False):
```

Comment:
The word "image_align" is a bit vague. What about "align_images" or "align_floating_images"?